inlet/flow/decoder/netflow/decode.go

```diff
@@ -151,6 +151,8 @@ func (nd *Decoder) decodeRecord(version uint16, obsDomainID uint32, samplingRate
 		// Remaining
 		case netflow.NFV9_FIELD_FORWARDING_STATUS:
 			nd.d.Schema.ProtobufAppendVarint(bf, schema.ColumnForwardingStatus, decodeUNumber(v))
+		case netflow.NFV9_FIELD_FIRST_SWITCHED:
+			bf.TimeReceived = decodeUNumber(v) + sysOffset
```

Comment:
So, both additional options mean the same thing?

Reply:
not sure I get what you mean.

In the case of FIRST_SWITCHED, I use the field inside each of the flow in the flowset to set "TimeReceived", which is then used afterwards.

In the case of NetflowPacket, I use the timestamp on the netflow packet for all flows in the flowset.
The `ts` in that case is `ts = uint64(packetNFv9.UnixSeconds)`.

```
for _, fmsg := range flowMessageSet {
		if !nd.useTsFromFirstSwitched {
			fmsg.TimeReceived = ts
		}
		fmsg.ExporterAddress = exporterAddress
	}
```

In the case of the UDP timestamp, ts = timestamp from kernel (`uint64(in.TimeReceived.UTC().Unix())`).

Reply:
OK, I missed this part of the code.